nixos/modules/misc/mandoc.nix

```diff
@@ -17,13 +18,14 @@ in {
         default = [ "share/man" ];
         example = lib.literalExpression "[ \"share/man\" \"share/man/fr\" ]";
         description = lib.mdDoc ''
-          Change the manpath, i. e. the directories where
-          {manpage}`man(1)`
+          Change the paths included in MANPATH environment variable,
+          i. e. the directories where {manpage}`man(1)`
           looks for section-specific directories of man pages.
           You only need to change this setting if you want extra man pages
           (e. g. in non-english languages). All values must be strings that
           are a valid path from the target prefix (without including it).
-          The first value given takes priority.
+          The first value given takes priority. Note that this is not
+          adding manpath in {manpage}`man.conf(5)`.
```

Comment:
```suggestion
          The first value given takes priority. Note that this will not
          add manpath directives to {manpage}`man.conf(5)`.
```